packages/react/src/backlink/Backlink.tsx

```diff
@@ -29,7 +26,7 @@ function Backlink(props: BacklinkProps, ref: Ref<HTMLAnchorElement>) {
     <RACLink
       className={cx(
         className,
-        'group flex max-w-fit items-center gap-3 rounded-md p-2.5 no-underline focus:outline-none focus-visible:ring focus-visible:ring-black',
+        'group flex max-w-fit items-center gap-3 rounded-md p-2.5 no-underline focus:outline-none [&[data-focus-visible="true"]]:ring [&[data-focus-visible="true"]]:ring-black',
```

Comment:
Vi trenger faktisk ikke spesifisere `"true"`. Kan være like greit å fjerne det for å være consistent med hvordan det er gjort andre steder? https://github.com/code-obos/grunnmuren/blob/dd32311f84c60305eff6c080959eace1801eb7cb/packages/react/src/classes.ts#L21